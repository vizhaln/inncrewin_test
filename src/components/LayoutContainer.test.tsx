import React from 'react';
import { render } from '../../test/test-utils';

import LayoutContainer from './LayoutContainer';

it('can browse to the about page', async () => {
  // For `LayoutContainer`, you should be able to render the layout container, followed by navigating to the About page.
  const app = render(<LayoutContainer />);
  expect(app).toMatchSnapshot();
});
