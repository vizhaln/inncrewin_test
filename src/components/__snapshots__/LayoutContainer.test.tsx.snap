// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can browse to the about page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="c1fj04oo"
      >
        <div
          style="outline: none;"
          tabindex="-1"
        >
          <section>
            <h1
              class="tpact6h"
            >
              Welcome!
            </h1>
            <a
              href="/about"
            >
              Go to about
            </a>
            <div>
              <p>
                Looks good
              </p>
            </div>
          </section>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="c1fj04oo"
    >
      <div
        style="outline: none;"
        tabindex="-1"
      >
        <section>
          <h1
            class="tpact6h"
          >
            Welcome!
          </h1>
          <a
            href="/about"
          >
            Go to about
          </a>
          <div>
            <p>
              Looks good
            </p>
          </div>
        </section>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "history": Object {
    "_onTransitionComplete": [Function],
    "listen": [Function],
    "location": Object {
      "hash": undefined,
      "host": undefined,
      "hostname": undefined,
      "href": undefined,
      "key": "initial",
      "origin": undefined,
      "pathname": "/",
      "port": undefined,
      "protocol": undefined,
      "search": "",
      "state": null,
    },
    "navigate": [Function],
    "transitioning": false,
  },
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
